lib/plugins/physics.js

```diff
@@ -220,6 +223,48 @@ function inject (bot) {
     return vel.x * vel.x + vel.z * vel.z
   }
 
+  function liquidPush(pos){
+    let flow = liquidFlow(pos);
+    let push = flow.clone().scaled(physics.liquidPush);
+    push.y = push.y/physics.gravity;
+    console.log(flow);
+    return push;
+  }
+
+  function liquidFlow(pos) {
+    let flow = new Vec3(0,0,0);
+    let base = bot.blockAt(pos).metadata;
+    let id = bot.blockAt(pos).type;
```

Comment:
add `const block = bot.blockAt(pos)` and use it instead of calling blockAt multiple times